### PR TITLE
[Container Registry] Test fixes

### PR DESCRIPTION
**Pull request by seankane-msft**
Fixing the environment variables in tests to work with the pipeline.

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

File: sdk/containerregistry/azure-containerregistry/samples/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ products:
 urlFragment: containerregistry-samples
 ---
 
-These code samples show common scenario operations with the Azure Container Registry client library.
+These code samples show common scenario operations with the Azure Container Registry client library. The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` set including the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).
```
Comment (catalinaperalta) [inline sdk/containerregistry/azure-containerregistry/samples/README.md:13]: Not sure if this is just me or not but I think this sentence should be like:
```
The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` is set, which includes
```

**Review by catalinaperalta — APPROVED** [review]

File: sdk/containerregistry/azure-containerregistry/samples/README.md (hunk 1 above)
Comment (seankane-msft) [inline sdk/containerregistry/azure-containerregistry/samples/README.md:13]:
```suggestion
The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` is set, which includes the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).
```

Comment (seankane-msft) [reply sdk/containerregistry/azure-containerregistry/samples/README.md]: Updated my poor grammar, thanks Catalina!

**Pull request merged**